dom-event-delegation/main.js

```diff
@@ -0,0 +1,12 @@
+var taskList = document.querySelector('.task-list');
+
+taskList.addEventListener('click', taskCheck);
+
+function taskCheck(event) {
+  console.log('target:', event.target);
```

Comment:
You're missing a console log for the `event.target.tagName`

Reply:
![target-tag](https://user-images.githubusercontent.com/118767169/217898585-de12f2f1-f60c-45b2-893f-de0a890dcb79.png)